README.md

```diff
@@ -42,6 +42,7 @@ The current members of the Presto TSC are:
 | Rongrong Zhong            |[rongrong](https://github.com/rongrong)              | Facebook    | 1 Jan, 2020  | TBD          |
 | Shixuan Fan               |[shixuan-fan](https://github.com/shixuan-fan)        | Facebook    | 1 Jan, 2020  | TBD          |
 | Jiexi Lin                 |[jessesleeping](https://github.com/jessesleeping)    | Facebook    | 1 Jan, 2020  | TBD          |
+| Venki Korukanti           |[vkorukanti](https://github.com/vkorukanti)          | Uber        | 14 Feb, 2020  | TBD         |
```

Comment:
Given that TSC approved adding Venki, this can be a separate PR and can be merged. The rest of the PR is more complicated and may need more time to review.